CONTRIBUTING.md

```diff
@@ -0,0 +1,80 @@
+# Contributing
+
+- [Contributing](#contributing)
+  - [GitHub Flow](#github-flow)
+    - [How to work with GitHub Flow](#how-to-work-with-github-flow)
+  - [Working with Forks and Pull Requests](#working-with-forks-and-pull-requests)
+  - [Working with Issues / How to Contribute in Issues](#working-with-issues--how-to-contribute-in-issues)
+  - [How to Write Commit Messages](#how-to-write-commit-messages)
+  - [Code Review](#code-review)
+  - [Code of Conduct](#code-of-conduct)
+  - [Code Style](#code-style)
+  - [Contributor License Agreement (CLA)](#contributor-license-agreement-cla)
+
+## GitHub Flow
+
+For more information see [GitHub Flow](https://githubflow.github.io/)
+
+GitHub Flow is very lightweight (especially compared to GitFlow).
+This workflow uses only two kinds of branches:
+
+- Feature branch
+- Main branch (previously called master)
+
+The `feature` branches are used to develop new features as well as fixes.
+These branches are usually created out of main.
+
+Anything in the `main` branch is deployable.
+The `main` branch is expected to be deployed regularly and is considered stable.
+
+### How to work with GitHub Flow
+
+For more information see [GitHub Flow](https://githubflow.github.io/)
+
+TL;DR
+
+1. anything in the main branch is deployable
+2. create descriptive branches off of main
+3. push to named branches constantly
+4. open a pull request at any time
+5. merge only after pull request review
+6. deploy immediately after review
+
+## Working with Forks and Pull Requests
+
+Our current continuous integration workflow makes it very hard to work on own forks of the SecureCodeBox because CI tests cannot be executed outside of our repository. We are aware of that problem and are working on a solution.
+
+Generally the SCB project follows the standard [GitHub Pull request process](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests).
+
+## Working with Issues / How to Contribute in Issues
+
+For every task taking more than 1h, opening an issue is required.
```

Comment:
I suggest the phrase: It is mandatory to open an issue, if take longer than one hour.